### DISCUSSION TITLE
Error installing liionpack

### DISCUSSION THREAD

**Discussion by Azoun83** [pybamm-team/liionpack#307] [Q&A]
Hi,
I receive the following error during liionpack installation after installing pybamm 24.1:

`The conflict is caused by:
liionpack 0.3.10 depends on ray
liionpack 0.3.9 depends on ray
liionpack 0.3.8 depends on pybamm==23.9
liionpack 0.3.7 depends on ray
liionpack 0.3.6 depends on ray
liionpack 0.3.5 depends on ray
liionpack 0.3.3 depends on ray
liionpack 0.3.2 depends on ray
liionpack 0.3.1 depends on ray
liionpack 0.3 depends on ray`

Can you help?

Thanks in advance.

**Comment by TomTranter** [comment]
I suggest if you are using something like anaconda to make a new environment and do a fresh install

**Reply by Azoun83** [reply]
I am not using ANACONDA. I am using pip.

**Reply by TomTranter** [reply]
You could use virtualenv to make a new environment.
